frontend/src/app/components/homescreen/blocks/new-features.component.ts

```diff
@@ -30,7 +30,6 @@ import {Component} from '@angular/core';
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {DomSanitizer} from "@angular/platform-browser";
-import {dashboardWebsiteUrl} from "core-app/modules/dashboards/dashboard-constants.const";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/8ea9ef8aecb8f0ef4037ecd7f3818f19dab95be4/frontend/src/app/components/homescreen/blocks/new-features.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>